test/extensions/filters/http/jwt_authn/filter_test.cc

```diff
@@ -17,74 +17,106 @@ namespace Extensions {
 namespace HttpFilters {
 namespace JwtAuthn {
 
+class MockMatcher : public Matcher {
```

Comment:
Please move these mock classes to mock.h

Reply:
done.